CHANGELOG.md

```diff
@@ -60,6 +60,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Fixed bugs
 
+* `jj diff --stat` no longer panics when a filename is very long and there is
+  the right combination of adds and deletes in a file.
+
```

Comment:
nit: I don't think the broken version is released yet, so remove the changelog entry?